src/python/scripts/dx-fetch-bundled-depends

```diff
@@ -41,4 +41,4 @@ if 'DX_JOB_ID' in os.environ:
             if dep['id']['$dnanexus_link'].startswith('file-'):
                 print "*** Downloading bundled file", dep['name']
                 dxpy.download_dxfile(dep['id'], dep['name'])
-                subprocess.check_call(['dx-unpack', dep['name']])
+                subprocess.check_call(['dx-unpack', pipes.quote(dep['name'])])
```

Comment:
As we discussed, I think this is something that should be fixed in dx-unpack itself.